nevergrad/optimization/optimizerlib.py

```diff
@@ -1214,49 +1215,122 @@ def __init__(
         super().__init__(SplitOptimizer, locals())
 
 
+class ConfPortfolio(base.ConfiguredOptimizer):
```

Comment:
@teytaud I think this can support all Portfolio and Paraportfolio variant, including the ones using selection after some ratio of computation
I am thinking of renaming this configured optimizer to simply "Portfolio", which is a breaking change, but that seems more logical.

Reply:
I'll update all the related optimizers in a followup diff